pkg/controller/machine/machine_controller.go

```diff
@@ -305,3 +341,35 @@ func (r *ReconcileMachine) deleteNode(ctx context.Context, cluster *clusterv1.Cl
 
 	return corev1Remote.Nodes().Delete(name, &metav1.DeleteOptions{})
 }
+
+// getMachines returns all of the Machine objects that belong to the same
+// cluster as the provided Machine
+func (r *ReconcileMachine) getMachines(ctx context.Context, machine *clusterv1.Machine) ([]*clusterv1.Machine, error) {
+	clusterName := machine.Labels[clusterv1.MachineClusterLabelName]
+	if clusterName == "" {
+		klog.Infof("Machine %q in namespace %q doesn't specify %q label, assuming nil cluster", machine.Name, machine.Namespace, clusterv1.MachineClusterLabelName)
```

Comment:
We should remove this log line, as it's already logged out when getting the Cluster reference